src/MainWindow.vala

```diff
@@ -219,8 +219,32 @@ public class Tasks.MainWindow : Hdy.ApplicationWindow {
                 add_collection_source (collection_source);
             });
 
+            var last_selected_list = Application.settings.get_string ("selected-list");
+
             if (last_selected_list == "scheduled") {
                 listbox.select_row (scheduled_row);
+                listbox.row_selected (scheduled_row);
+
+            } else {
+                var default_task_list = registry.default_task_list;
+                var task_lists = registry.list_sources (E.SOURCE_EXTENSION_TASK_LIST);
+
+                task_lists.foreach ((source) => {
+                    E.SourceTaskList list = (E.SourceTaskList)source.get_extension (E.SOURCE_EXTENSION_TASK_LIST);
+
+                    if (list.selected == true && source.enabled == true) {
+                        add_source (source);
+
+                        if (last_selected_list == "" && default_task_list == source) {
+                            listbox.select_row (source_rows[source]);
+                            listbox.row_selected (source_rows[source]);
```

Comment:
`row_selected` should be emitted automatically when `select_row` is called. I don't think you need to forcefully emit it like this

Reply:
Removed the three calls - but now the Scheduled list is broken. To reproduce:

1. Compile from this branch and start Tasks
2. Select the Scheduled list
3. Exit the Tasks app
4. Start the Tasks app, now it looks like this:

![Screenshot from 2021-02-24 09-55-03](https://user-images.githubusercontent.com/392542/108975186-84c30280-7686-11eb-884c-3c6890c745eb.png)

... is this, because the "Scheduled" list is already selected on startup and therefore the `listbox.row_selected.connect` is never called? If so, how do you suggest to fix this?

Reply:
Fixed this by removing all `row_selected` apart from the one for the Scheduled list - don't know why its needed there, but now it works as intended.